### ISSUE TITLE
No pass reports with Config::Model 2.109

### ISSUE THREAD

**Issue by eserte** [dod38fr/config-model-itself#2]
On my smokers I see the following failure:
```
Prototype mismatch: sub CORE::GLOBAL::exit: none vs (;$) at /opt/perl-5.27.4/lib/site_perl/5.27.4/x86_64-linux/Tk.pm line 415.

#   Failed test 'edit is in test mode'
#   at t/cme-meta-edit.t line 37.
#                   'Reading model from /opt/perl-5.27.4/lib/site_perl/5.27.4/Config/Model
# '
#     doesn't match '(?^:Test mode: quit)'
# Looks like you failed 1 test of 2.
t/cme-meta-edit.t .......... 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/2 subtests 
Prototype mismatch: sub CORE::GLOBAL::exit: none vs (;$) at /opt/perl-5.27.4/lib/site_perl/5.27.4/x86_64-linux/Tk.pm line 415.

#   Failed test 'threw no exceptions'
#   at t/cme-meta-plugin.t line 58.
#          got: 'In class:Fstab: (function 'create_element') unknown element 'rw_config'. Either your file has an error or Itself::Class model is lagging behind. In the latter case, please submit a bug report to https://github.com/dod38fr/config-model/issues. See cme man page for details.
# 	Expected elements: 'class_description','license','author','copyright','class','element','include','include_backend','include_after','generated_by','read_config','write_config','accept'
# 	
# '
#     expected: undef
... (etc.) ...
```

Statistical analysis shows at least that there are no pass reports if Config::Model 2.109 is installed:
```
****************************************************************
Regression 'mod:Config::Model'
****************************************************************
Name           	       Theta	      StdErr	 T-stat
[0='const']    	      1.0000	      0.0314	  31.85
[1='eq_2.104'] 	     -1.0000	      0.0415	 -24.07
[2='eq_2.105'] 	     -0.0149	      0.0334	  -0.45
[3='eq_2.106'] 	      0.0000	      0.0378	   0.00
[4='eq_2.108'] 	      0.0000	      0.0475	   0.00
[5='eq_2.109'] 	     -1.0000	      0.0736	 -13.58

R^2= 0.925, N= 117, K= 6
****************************************************************
```

**Comment by dod38fr** [comment]
Bummer. The automatic translation of old model parameters (read_config) into new ones (rw_config) does break Config::Model::Itself.  I did not foresee that.

I'll release a new version of Config::Model::Itself real soon.

Thanks for the report

**Comment by dod38fr** [comment]
On second thought, this breakage also impacts the nice folks at RedHat who package Config::Model and co. I'd better backpedal a little with the deprecation done in Config::Model

**Comment by dod38fr** [comment]
Fixed with Config::Model 2.110

Thanks for the report
